### PR TITLE
scons, new python project and run python project

### PR DESCRIPTION
**Pull request by ShiCheng-Lu**

### CODE REVIEW

**Comment by ShiCheng-Lu** [comment]
Changed: files in scons folder, files in py folder, Sconsconstruct

File: scons/build.scons [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import os
+
+Import("VARS")
+
+TYPE = VARS.get("TYPE")
+TARGET = VARS.get("TARGET")
+PLATFORM = VARS.get("PLATFORM")
+env = VARS.get("env")
+
+BUILD_DIR = Dir('#/build').Dir(PLATFORM)
```
Comment (JarvisWeng) [inline scons/build.scons:12]: Can you change to this instead Dir('#').Dir(.....)

File: scons/build.scons [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                target.append(header_dir.File(template.name[:-6]))
+    
+    source.append(source_yaml)
+    env.Command(header_files, source, header_command)
+    env.Command(source_files, source, source_command)
+
+    target += source_files + header_files
+    return source_files, header_dir
+
+def addBuildPath(string: str):
```
Comment (JarvisWeng) [inline scons/build.scons:108]: You should not need this function

File: scons/build.scons [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    srcs = OBJ_DIR.Dir(str(entry)).Dir('src').glob('*.[cs]')
+    srcs += OBJ_DIR.Dir(str(entry)).Dir('src').Dir(PLATFORM).glob('*.[cs]')
+    
+    inc_dirs = [entry.Dir('inc')]
+    inc_dirs += [entry.Dir('inc').Dir(PLATFORM)]
+    
+    config = parse_config(entry)
+    
+    if config["can"]:
+        # TODO: Current output files are like so
```
Comment (JarvisWeng) [inline scons/build.scons:128]: Wanna fix this for me :)

**Review by JarvisWeng — APPROVED** [review]

**Pull request merged**